pkg/api/admin_provisioning.go

```diff
@@ -0,0 +1,30 @@
+package api
+
+import (
+	"context"
+	m "github.com/grafana/grafana/pkg/models"
```

Comment:
This is a weird convention that is used in Grafanas codebase. 
It's not something we should continue doing. The only reason for aliasing imports is when the imported package name contains a dash. I'll bring this up in the next weekly meeting and make sure to add it to our style guide.